### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by AntonTrollback**
Mostly fixing typos

### CODE REVIEW

**Comment by wnr** [comment]
Sweet! Will read and merge asap.

File: README.md [hunk 1]
```diff
@@ -1,39 +1,38 @@
...
-# Docs
+# Documentation
 
-Since ELQ is in beta, we haven't had time to write proper docs yet. What follows is really a minimal explanation of the API.
+We haven't had time to write proper docs yet. What follows is really a minimal explanation of the API.
 
 ## Usage example
 
-Once the ```dist/elq.js``` file is included into the HTML (it is built with UMD, so include it as you wish) it exposes a global function ```Elq```. This is a constructor that creates ELQ instances. It is recommended to only use one instance per application.
+Once the `elq.js` file is included intro the page (it is built with UMD, so include it as you wish) it exposes a global `Elq` constructor. It is recommended to only create one instance per application.
```
Comment (wnr) [inline README.md]: It should not be "intro" here right?

Comment (AntonTrollback) [inline README.md]: Whoops. I reverted the paragraph, the original is a bit clearer

**Comment by wnr** [comment]
Lookin' good.

**Pull request merged**